src/sshping.cxx

```diff
@@ -22,6 +22,13 @@
    SOFTWARE.
  */
 
+#ifdef _WIN32
+#define LIBSSH_STATIC 1
```

Comment:
Nit style thing: I indent conditionalized preprocessor directives 2 spaces (see for example the LIBSSH version check a few lines below)